src/components/field/index.tsx

```diff
@@ -1,12 +1,19 @@
+
 import {ExpandedType} from 'compassql/build/src/query/expandedtype';
+import {Schema} from 'compassql/build/src/schema';
 import {isWildcard} from 'compassql/build/src/wildcard';
 import * as React from 'react';
 import * as CSSModules from 'react-css-modules';
 import {DragElementWrapper, DragSource, DragSourceCollector, DragSourceSpec} from 'react-dnd';
+import {OneOfFilter, RangeFilter} from 'vega-lite/build/src/filter';
+
+import * as styles from './field.scss';
```

Comment:
I just learned recently that having no line breaks between imports makes it easier for autoformatter to fix things without causing merge confilcts -- so let's do that going forward.


cc: @felixcodes